apps/survey/package.json

```diff
@@ -17,7 +17,15 @@
     "prepare": "npx apps-shared-scripts-prepare",
     "install:frontend": "cd app && npm install",
     "abi:extract": "truffle-extract --output abi/ --keys abi",
-    "prepublishOnly": "truffle compile --all && npm run abi:extract -- --no-compile"
+    "prepublishOnly": "truffle compile --all && npm run abi:extract -- --no-compile",
+    "publish:kit:rpc": "aragon apm publish major $(npm run deploy:kit:rpc | tail -n 1) --environment default",
+    "deploy:kit:rpc": "aragon contracts compile --all && aragon contracts exec --network rpc scripts/ deploy.js",
+    "publish:kit:staging": "aragon apm publish major $(npm run deploy:kit:staging | tail -n 1) --environment staging",
+    "deploy:kit:staging": "aragon contracts compile --all && ENS=0xfe03625ea880a8cba336f9b5ad6e15b0a3b5a939 aragon contracts exec --network rinkeby scripts/ deploy.js",
+    "publish:kit:rinkeby": "aragon apm publish major $(npm run deploy:kit:rinkeby | tail -n 1) --environment rinkeby",
+    "deploy:kit:rinkeby": "aragon contracts compile --all && ENS=0x98Df287B6C145399Aaa709692c8D308357bC085D aragon contracts exec --network rinkeby scripts/ deploy.js",
+    "publish:kit:mainnet": "aragon apm publish major $(npm run deploy:kit:mainnet | tail -n 1) --environment mainnet",
+    "deploy:kit:mainnet": "aragon contracts compile --all && ENS=0x314159265dd8dbb310642f98f50c066173c1259b aragon contracts exec --network mainnet scripts/ deploy.js"
```

Comment:
It would be nice to not have most of these npm scripts, since the example kit is unlikely to be published very many times by us.

cc @0xGabi which scripts does aragen depend on?

Reply:
Sure, I honestly don't know how they are used and if they are used in a certain way actually. I migrated all the scripts related stuff that was in the survey kit repo. Let me know if I should remove those

Reply:
aragen is only using the `publish:kit:rpc` and `deploy:kit:rpc` but I believe it would be posible to use aragonCLI directly in the bash file ([here](https://github.com/aragon/aragen/blob/master/scripts/publish-templates#L13)) 

My only unknown is how to reference the different template deploy scripts, I believe something like `$SCRIPT_NAME`.

Reply:
> My only unknown is how to reference the different template deploy scripts, I believe something like $SCRIPT_NAME.

Hmm, do you mean which script aragen would use? Perhaps we can standardize on a `deploy_kit.js` or something that will be used by aragen?

Reply:
Yes exactly. You are right, we should standardize. Will check if that's possible with current `deploy_kit.js`